sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -56,6 +60,24 @@ internal class RadarActivityLifecycleCallbacks : Application.ActivityLifecycleCa
         foreground = count > 0
 
         updatePermissionsDenied(activity)
+
+        Radar.logConversion(
+            "app_open",
+            null,
+            null,
+            null,
+            null,
```

Comment:
having four nulls in a row is giving me a smell of bad code. With your greater familiarity around android, does this smell of bad code to you too? Can we set the default values in the `internal fun logConversion` to be null so we don't need to pass null 4 times?

Reply:
yeah, this is definitely a cleaner solution